internal/cli/atlas/organizations/create_atlas.go

```diff
@@ -126,7 +126,8 @@ func CreateAtlasBuilder() *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:   "create <name>",
-		Short: "Create an Ops Manager or Cloud Manager organization. This command is unavailable for Atlas.",
+		Short: "Create an organization.",
+		Long:  "When authenticating using API keys the requesting keys most have cross organization billing enable and the resulting org will be linked to the paying org.",
```

Comment:
Question: does the API key have cross-organization billing enabled? Or should this say "the organization to which the API keys belong must have cross-organization billing enabled"?
```suggestion
		Long:  "When authenticating using API keys, the organization to which the API keys belong must have cross-organization billing enabled. The resulting org will be linked to the paying org.",
```

Reply:
@sarahsimpers you are correct, I've updated it